astropy/visualization/wcsaxes/ticklabels.py

```diff
@@ -180,6 +180,10 @@ def draw(self, renderer, bboxes, ticklabels_bbox):
 
                     dist = np.hypot(dx, dy)
 
+                    # Avoid RuntimeWarning
+                    if dist == 0 or not np.isfinite(dist):
```

Comment:
I couldn't tell from the Travis log where the `RuntimeWarning` came from. I built the doc locally and did not see any difference in the generated images in the "Example gallery". Maybe I am looking in the wrong place? But regardless we need to address this one way or another, since `ddx` and `ddy` will not be a real number anyway. I am just not sure whether `ddx = dx` (in this PR) or `ddx = 0` is more logical.